src/Microsoft.ML.FastTree/GamRegression.cs

```diff
@@ -107,44 +107,44 @@ protected override SchemaShape.Column[] GetOutputColumnsCore(SchemaShape inputSc
         }
```

Comment:
unrelated to your PR, but since i stumped upon this, is this used at all?

Reply:
I guess it is part of `TrainerEstimatorBase` and has to be implemented

---
In reply to: [243714960](https://github.com/dotnet/machinelearning/pull/1901#discussion_r243714960) [](ancestors = 243714960)